engine.Dockerfile

```diff
@@ -39,6 +39,8 @@ RUN <<-```
     curl -sSL https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-cli-516.0.0-linux-x86_64.tar.gz -o /tmp/gcloud.tar.gz
     mkdir -p /opt && tar -C /opt -xf /tmp/gcloud.tar.gz
     /opt/google-cloud-sdk/install.sh --quiet
+    /opt/google-cloud-sdk/bin/gcloud components install beta --quiet
```

Comment:
why did you need this? vs the previous `/opt/google-cloud-sdk/install.sh` ?

Reply:
The recommenders list cli call needs to be done with goole cloud beta, and you have to install it as a component